js/controller.js

```diff
@@ -396,6 +399,18 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         //adVideoDuration is only used in adPanel ad marquee
         this.state.adVideoDuration = duration;
       }
+
+      // lower skin z-index if Chrome auto-pauses flash content
+      if(!this.state.autoPauseDisabled && Utils.isChrome && this.state.mainVideoMediaType != CONSTANTS.MEDIA_TYPE.HTML5) {
+        var skinElement = $("#"+this.state.elementId+" .oo-player-skin");
+        if(currentPlayhead == 0 && this.state.playerState == CONSTANTS.STATE.PLAYING) {
```

Comment:
Is this the state the player ends up in when auto paused? Not sure I understand playhead == 0 while playing.